pkg/http/logging/logging.go

```diff
@@ -0,0 +1,62 @@
+package logging
```

Comment:
The `pkg/http/` subdirectory is unnecessary and redundant in a repository called `go-http-middleware`. The http middleware packages should simply live at the root of the repo.

Reply:
That makes sense.